basic_ui/ClosingScreen.qml

```diff
@@ -51,8 +51,10 @@ Rectangle {
     onStartAnimFinishedChanged: {
         if (startAnimFinished) {
             console.debug("NOW SHUTDOWN");
+            // TODO create a framebuffer device class instead of launching hard coded shell scripts from QML
```

Comment:
Image should be visible after the app is closed. It should not clear the framebuffer when the app is closed. Just a note.

Reply:
The default implementation will simply call the current shell script. The initial focus is to remove all shell script launches from qml to have a flexible design.
This will be implemented in #346 Hardware device abstraction.